core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-publish-variant/dot-experiments-publish-variant.component.scss

```diff
@@ -1,5 +1,20 @@
+@use "variables" as *;
+
+.control {
+    margin-left: $spacing-1;
+}
+
 :host ::ng-deep {
     .p-datatable .p-datatable-tbody > tr {
         cursor: default;
+        transition: background-color $basic-speed ease-in-out;
+
+        td {
+            border: none;
+        }
+
+        &:hover {
+            background-color: $color-accessible-text-gray-bg;
```

Comment:
Do we need to add `rowHover` to the table and move this to the global styles?

Reply:
I've fixed it as you suggested. I fixed the hover color.